dz_7/dz_7_1.py

```diff
@@ -9,6 +9,7 @@
 # **Ввод:** пара-ра-рам рам-пам-папам па-ра-па-да    
 #     **Вывод:** Парам пам-пам 
 
+
 def ritm(texts):
     words = texts.split()
     count_glasnyi = lambda word: sum(1 for letter in word if letter.lower() in 'аеёиоуыэюя')
```

Comment:
Можно иначе
```suggestion
    words = texts.lower().split()
    count_glasnyi = lambda word: sum(letter in 'аеёиоуыэюя' for letter in word)
```
Подумайте, что я  сделал